internals/secrethub/signup.go

```diff
@@ -181,7 +185,42 @@ func (cmd *SignUpCommand) Run() error {
 	}
 
 	cmd.progressPrinter.Stop()
+	fmt.Fprintln(cmd.io.Stdout(), "Created your account.")
+
+	createWorkspace := cmd.org != ""
+	if !createWorkspace {
+		createWorkspace, err = ui.AskYesNo(cmd.io, "Do you want to create a shared workspace for your team?", ui.DefaultYes)
+		if err != nil {
+			return err
+		}
+		if !createWorkspace {
+			fmt.Fprintln(cmd.io.Stdout(), "You can create a shared workspace later using `secrethub org init`.")
+		}
+	}
+	if createWorkspace {
+		if cmd.org == "" {
+			cmd.org, err = ui.AskAndValidate(cmd.io, "Workspace name for your team (e.g. your company name): ", 2, api.ValidateOrgName)
```

Comment:
the `for your team` part is a bit redundant as it's already stated in the question above `do you want to create a shared workspace for your team?`. Removing it here makes it a bit shorter:

`Workspace name (e.g. your company name):`